src/index.js

```diff
@@ -1,2 +1,37 @@
 import './style.css';
-import './modules/Player.js';
\ No newline at end of file
+import API from './modules/API.js';
+
+const nameEl = document.getElementById('your-name');
+const scoreEl = document.getElementById('your-score');
+const btnReferesh = document.querySelector('.btn-refresh');
+const btnSubmit = document.querySelector('.btn-submit-score');
+const scoresList = document.querySelector('.scores-info');
+const api = new API();
+
+// display scores
+const displayScores = async () => {
+  const scoresArray = await api.getScores();
+  let scoresData = '';
+  const sortedScores = scoresArray.result.sort((a, b) => b.score - a.score);
+  sortedScores.forEach((item) => {
+    scoresData += `<li>${item.user}: ${item.score}</li>`;
+  });
+  scoresList.innerHTML = scoresData;
+};
+
+// refresh button
+btnReferesh.addEventListener('click', async () => {
+  displayScores();
+});
+
+btnSubmit.addEventListener('click', async (e) => {
+  e.preventDefault();
+  const playerName = nameEl.value;
+  const playerScore = scoreEl.value;
+  await api.addScore(playerName, playerScore);
+  displayScores();
```

Comment:
- [ ] @AbeeraTahir great work on your app I loved it however, the project requirements require separation of saving and score list refresh, and your current implementation automatically refreshes the page.
![image](https://user-images.githubusercontent.com/61408860/202263622-c8464ae7-6ca1-41d7-b046-20a9940f9898.png)